rxjava-support/src/test/java/tech/picnic/rx/PicnicRxPluginsTest.java

```diff
@@ -50,14 +50,15 @@ public void testPropagate() {
     verifyVerificationCounter(ctx2, 3);
 
     // With context propagation
-    init();
+    setUp();
     obs.subscribeOn(Schedulers.io())
         .doOnNext(i -> verifyActive(ctx1))
         .ignoreElements()
         .blockingAwait();
     verifyVerificationCounter(ctx1, 3);
   }
 
+  @Test
```

Comment:
Shame we lose class level annotations for `Test`

Reply:
Yep.